packages/torghostng/PKGBUILD

```diff
@@ -0,0 +1,40 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=torghostng
+pkgver=119.348f639
+pkgrel=1
+pkgdesc='TorghostNG is a tool that make all your internet traffic anonymized through Tor network.'
+url='https://github.com/GitHackTools/TorghostNG'
+arch=('any')
+groups=('blackarch' 'blackarch-proxies')
+license=('GPL3')
+depends=('python' 'tor' 'macchanger' 'privoxy' 'curl' 'net-tools')
+makedepends=('git')
+source=('git+https://github.com/GitHackTools/TorghostNG.git')
+sha512sums=('SKIP')
+
+pkgver(){
+  cd TorghostNG
+  echo $(git rev-list --count HEAD).$(git rev-parse --short HEAD)
+}
+
+package(){
+  cd TorghostNG
+
+  install -dm 755 "$pkgdir/usr/bin/"
+  install -dm 755 "$pkgdir/usr/share/$pkgname"
+
+  install -Dm 644 -t "$pkgdir/usr/share/doc/$pkgname/" *.md
+  install -Dm 644 LICENSE "$pkgdir/usr/share/licenses/$pkgname/LICENSE"
```

Comment:
[Don't copy the license for GPL](https://wiki.archlinux.org/title/PKGBUILD#license)